test/test_binary_ufuncs.py

```diff
@@ -2495,6 +2499,55 @@ def _promo_helper(x, y):
                     with self.assertRaisesRegex(RuntimeError, "is not the desired type"):
                         torch.Tensor.float_power_(base.clone(), exp)
 
+    @skipIf(not TEST_SCIPY, "Scipy required for the test.")
+    @dtypes(*product(torch.testing.get_all_dtypes(include_complex=False, include_bfloat16=False),
```

Comment:
xlogy is implemented for bfloat16, though

Testing a torch function that supports bfloat16 against a NumPy (or SciPy) reference is, unfortunately, a little tricky. It'd be nice if we had more helpers to support it. 

In this case, a special test for xlogy in bfloat16 could be added. It does not need to be as thorough as this test. For example, it does not need to test that out works properly, but this test (or another test) should verify that.

Reply:
Added a new function for it.